backend/LexBoxApi/Controllers/AdminController.cs

```diff
@@ -36,6 +39,7 @@ public async Task<ActionResult> ResetPasswordAdmin(ResetPasswordAdminRequest req
         var passwordHash = request.PasswordHash;
         var user = await _lexBoxDbContext.Users.FirstAsync(u => u.Id == request.userId);
         user.PasswordHash = PasswordHashing.HashPassword(passwordHash, user.Salt, true);
+        await _userService.SetPasswordStrength(user.Id, request.PasswordStrength);
```

Comment:
is there a reason we don't just do `user.PasswordStrength = request.PasswordStrength`?

Reply:
The SetPassswordStrength method clamps the number to the range [0..4]. But I could just as easily write a ClampPasswordStrength method and call it, then do `user.PasswordStrength = Clamp(request.PasswordStrength)` here and save a DB round-trip.

Reply:
Done in commit 9ca48737.